### PR TITLE
create separate component for opt popups, fix styling in new popup

### PR DESCRIPTION
**Pull request by tickct**
So the issue is due to us using the same form for when we need a submit button, when we have text options and when we have opt options. 

To make this fix more long term. I have broken out popups of id OPT into a new form component and created and other form component to have a fallback. Moving forward this will allow us to do different things in the popup using the ID to define the layout style for the form.

image

image

Test Deck Link: https://fabrary.net/decks/01GX2F3RM59TYDSWFK1T92MFCY

### CODE REVIEW

**Comment by LaustinSpayce** [comment]
I left some comment / change which needs to be addressed otherwise all LGTM and I can merge it after the change is done.

**Comment by tickct** [comment]
I don't see any comments on this PR at the moment, happy to adjust once they are here

**Review by LaustinSpayce — CHANGES_REQUESTED** [review]

**Comment by LaustinSpayce** [comment]
> I don't see any comments on this PR at the moment, happy to adjust once they are here

Turns out I need to start it, and then press "request changes" so you can see them, my bad.

**Review by LaustinSpayce — APPROVED** [review]
lgtm thanks for making the change

**Pull request merged**
